README.md

```diff
@@ -2,7 +2,9 @@
 
 Admin interface for managing Local Authorities' links including all their services and interactions.
 
-For example, [this service](https://www.gov.uk/garden-waste-disposal) uses Local Links Manager to provide the URL on a Local Authority's website that contains information on disposing of garden waste.
```

Comment:
Did you mean to remove this link?

Reply:
I didn't. Thanks. Is there a way I can accept your suggestion, or do I need to make another new commit?

Reply:
You can accept my suggestion if I mark it up in the comment (using '''suggestion syntax, but with ` instead of '). I'll do that here:

```suggestion
For example, [this service](https://www.gov.uk/garden-waste-disposal) uses Local Links Manager to determine the URL of the garden waste disposal page on the Local Authority website which corresponds to the postcode a user has entered.
```

But as this is a fixup commit (correcting a mistake that you don't really need to retain a history of), it's probably better to follow the CLI instructions for amending the commit and writing over your branch, rather than creating a new commit. Depends how far you'd like to take this learning session!

Reply:
I've discussed with @ChrisBAshton and will make a new commit, as I don't plan to learn how to amend commits at this early stage in my Git learning journey 🛤